### PR TITLE
Fix GCC and Windows CI Tests

### PR DESCRIPTION
**Pull request by stefankiesz**
### Mac GCC CI Test ###
Changed to using a direct and versioned compiler path - just pointing to "gcc" and "g++" will map to CLANG compiler on Mac.

<br>

### Windows MSVC CI Test ###
Updated PKG_CONFIG_EXECUTABLE to reflect current Chocolatey GSTreamer install path - it is  in D: drive now, not C: drive.

<br>
<br>
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -127,7 +127,7 @@ else()
 endif()
 
 if (WIN32)
-  set(PKG_CONFIG_EXECUTABLE "C:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
+  set(PKG_CONFIG_EXECUTABLE "D:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
```
Comment (niyatim23) [inline CMakeLists.txt:130]: If this path is specific to the CI, should we make allowing customizing it? What was this path on your local Windows machine (if you were able to try this)?

Comment (stefankiesz) [reply CMakeLists.txt:130]: On local machine it is C:, I think best way to handle this is to specify the location for Chocolatey installs so we can use C: for everything, unless customers are finding that it sometimes does get directed to D: for them as well. Going to look into this now.

Comment (niyatim23) [reply CMakeLists.txt:130]: Okay, we can keep it D:\. Let's a note in the readme for this, mentioning that the customers can change this if this path is different in their installation

Comment (stefankiesz) [reply CMakeLists.txt]: I tried setting gstreamer installation path using by passing an install argument to choco command, and that resulted in neither the C nor D drive having the gstreamer installation. I ended up putting a check in CMake to see if the C drive path exists, else use D drive.

**Review by niyatim23 — APPROVED** [review]
Nice work on fixing the both!

**Pull request merged**